tests/lib/notify/testcase.yaml

```diff
@@ -0,0 +1,3 @@
+tests:
+  libraries.sys_notify:
+    tags: sys_notify
```

Comment:
You could also add here `notify` tag (easier to remember)

Reply:
replaced